.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,14 @@
+blank_issues_enabled: false
+contact_links:
+  - name: PowerShell Issues
```

Comment:
Are these applicable to this repository, or should they be customized for Crescendo?

Reply:
These are _intended_ for users who find a problem with PowerShell/Windows PowerShell itself and should report the problem there (instead of reporting it here). I can clarify that if needed.